web/src/pages/BlockList/index.css.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components'
+
+export const BlockListPanel = styled.div`
+  width: 100%;
+  margin-top: ${(props: { width: number }) => (150 * props.width) / 1920}px;
+  margin-bottom: ${(props: { width: number }) => (200 * props.width) / 1920}px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+export const ContentTitle = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  div {
+    font-size: 50px;
+    color: black;
+    margin: 0 auto;
+  }
+
+  span {
+    background: #46ab81;
+    height: 4px;
+    width: 197px;
+    margin: 0 auto;
+  }
+`
+
+export const ContentTable = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-top: 58px;
+`
+
+export const TableTitleRow = styled.tr`
+  background: rgb(75, 188, 142);
+  display: flex;
```

Comment:
tr tag is default as row for its td-child-elements, why keep display: flex, flex-direction: row,  here?

Reply:
Without flex, it will show vertically

Reply:
it is a weird layout for table tr, I never use it before. by default, I think tr(table-row) is in a row for a table. maybe here there is no need to use a table, looks like a div if you set the display with flex in tr/td to change the behavior

Reply:
OK. I use `div` to implement.